app/models/host/managed.rb

```diff
@@ -1,5 +1,19 @@
 class Host::Managed < Host::Base
+  cattr_accessor :cloned_parameters
+  self.cloned_parameters = {}
+
+  def self.include_in_clone(*attributes)
```

Comment:
@ShimShtein How about extracting include/exclude to a module that can be also used in Hostgroup? That would be a first, easy to merge PR. Also instead of initializing cloned_parameters to an empty hash, `Hash.new([])` sets the default so that these two methods could just be `cloned_parameters[:include] += attributes`